web/src/components/organisms/Project/Content/ContentDetails/index.tsx

```diff
@@ -15,10 +18,35 @@ const ContentDetails: React.FC = () => {
     handleItemUpdate,
     handleNavigateToModel,
   } = useHooks();
+  const { projectId } = useParams();
+  const {
+    assetList,
+    handleSearchTerm,
+    handleAssetsReload,
+    loading,
+    createAssets,
+    fileList,
+    setFileList,
+    setUploading,
+    setUploadModalVisibility,
+    uploading,
+    uploadModalVisibility,
```

Comment:
I already asked @issmail-basel to do this too. Can prop order/hook return order be variables followed by functions? And of course, variables related to one another should be together and functions related to one another should be together.

Reply:
Please correct me if I am wrong, 
The order of the arguments(variables, methods) should be by:
1- Variables then methods.
2- Grouped by relativity to on another.
3- Alphabetically.
right?

Reply:
In this case the order should be like this:
```js
  const {
    assetList,
    fileList,
    loading,
    uploading,
    uploadModalVisibility,
    createAssets,
    handleSearchTerm,
    handleAssetsReload,
    setFileList,
    setUploading,
    setUploadModalVisibility,
```

Reply:
Ah you don't need to worry about alphabetical order! But the other points are what I hope we can aim for as much as possible! 

Also, keeping the return order from the hooks. For example, if the hooks.ts file is:
```
return {
    assetList,
    fileList,
    loading,
    uploading,
    uploadModalVisibility,
}
```
then we just copy paste that order where its being used:
const {
    assetList,
    fileList,
    loading,
    uploading,
    uploadModalVisibility,
} = useHooks();

Reply:
If this seems like too much or annoying or anyways you just are against this approach, please fight me on it :P But for now, this is what I hope for.

Reply:
It is not too much at all. it is cleaner this way.
No need to fight you on it ;)
Done